### PR TITLE
Sync Changes

### LINKED ISSUE #6: Remove potential security vulnerabilities in the dependencies
Probably we have old package json lock file. Maybe we should recreate it

### LINKED ISSUE #9: Upgrade React component to latest release.
React have new full release 18.0
https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html

### LINKED ISSUE #10: I think it's not good to have logic and jsx elements in one module. Much better will be split several modules
I think it's not good to have logic and jsx elements in one module. Much better will be split several modules

_Originally posted by @Lehersun in https://github.com/skbu3u/web-interface/pull/8#discussion_r840742092_

### LINKED ISSUE #11: I see many codestyle problems, as well a lot of commented parts and development feature like console.log. Would be nice to set codeSniffer for JS for automize watching for codestyle https://www.npmjs.com/package/jscodesniffer
I see many codestyle problems, as well a lot of commented parts and development feature like console.log. Would be nice to set codeSniffer for JS for automize watching for codestyle https://www.npmjs.com/package/jscodesniffer

_Originally posted by @Lehersun in https://github.com/skbu3u/web-interface/pull/8#pullrequestreview-929175784_

### PR DESCRIPTION
**Pull request by xansochi**

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "Web Interface",
-  "name": "Create React Web Interface",
+  "short_name": "React App",
```
Comment (Lehersun) [inline public/manifest.json]: Why did you change name?

File: src/components/Login/Login.jsx [hunk 2]
```diff
@@ -1,64 +1,101 @@
...
 
   const submitHandler = (event) => {
     event.preventDefault();
-    props.onLogin(enteredEmail, enteredPassword);
+    props.onLogin(email, password)
   };
+  injectStyle();
+  const login = ()=> {
+
+    if(email==='' & password===''){
```
Comment (Lehersun) [inline src/components/Login/Login.jsx]: Would be nice if we'll trim these data before comparing

File: src/components/Login/Login.jsx [hunk 3]
```diff
@@ -1,64 +1,101 @@
...
+          }
+        })
+        .catch(function(error){
+          console.log(error,'error');
+          {notify()}
+        });
+    }
+}
 
   return (
```
Comment (Lehersun) [inline src/components/Login/Login.jsx:16]: I think it's not good to have logic and jsx elements in one module. Much better will be split several modules

**Review by Lehersun — COMMENTED** [review]
I see many codestyle problems, as well a lot of commented parts and development feature like console.log. Would be nice to set codeSniffer for JS for automize watching for codestyle https://www.npmjs.com/package/jscodesniffer

File: public/manifest.json (hunk 1 above)
Comment (xansochi) [reply public/manifest.json]: Because i create all wat u see in this branch how a new project. All files in old web-interface1 branch are useless. We can use any application name.What name do you prefer? :)

Comment (Lehersun) [reply public/manifest.json]: Would be nice if you will name it. It;s not a blank project, not a testing. It definataly a working project must be named

Comment (xansochi) [reply public/manifest.json]: ok :)

**Comment by xansochi** [comment]
Changelog
1 - ESLint 8.13.0 with React+JSX configuration. 
2 - Prettier 2.6.2 Now it work with ESLinter, but i need а correct configuration.
3 - Now we have notification system React Toastify. Works with API errors.
4 - All modal windows forms work correctly.There are 4 forms for working with the API, ADD Users, ADD Equipments, ADD parts, ADD Consumables.
5 - ADD parts and ADD Consumables forms contains a function for fetching data from a GET request and nonerror dependency addition to parent elements. Until we test it and there is no certainty that all this will be used in production, let logic and html in same file.
6 - Upgrade REACT to latest release 18.0.0
7 -  Add "smart" search module to dashboard :)
8 - A huge number of errors have been removed after a manual ESlinter pass. 
9 - Etc. :)

File: public/index.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html class="h-full bg-white-900" lang="en">
```
Comment (Lehersun) [inline public/index.html]: Why?

File: public/manifest.json [hunk 5]
```diff
@@ -1,6 +1,10 @@
 {
   "short_name": "Web Interface",
+<<<<<<< HEAD
```
Comment (Lehersun) [inline public/manifest.json]: Fix code after resolving conflicts

File: src/components/Pages/Equipments/EquipmentsPage.jsx [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+              >
+                Добавить оборудование
+              </button>
+              <ModalAddEq open={open} setOpen={setOpen} />
+            </div>
+          </div>
+        </div>
+        <div className="-my-2 overflow-x-auto sm:-mx-6 lg:-mx-8">
+          <div className="py-2 align-middle inline-block min-w-full sm:px-6 lg:px-8">
+            <div className="shadow overflow-hidden border-b border-gray-200 sm:rounded-lg">
```
Comment (Lehersun) [inline src/components/Pages/Equipments/EquipmentsPage.jsx:84]: https://ru.reactjs.org/docs/code-splitting.html Could we split components, for remove code duplicating

File: src/components/Pages/Parts/PartsPage.jsx [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+        <div className="-my-2 overflow-x-auto sm:-mx-6 lg:-mx-8">
+          <div className="py-2 align-middle inline-block min-w-full sm:px-6 lg:px-8">
+            <div className="shadow overflow-hidden border-b border-gray-200 sm:rounded-lg">
+              <table className="min-w-full divide-y divide-gray-200">
+                <thead className="bg-gray-50">
+                  <tr>
+                    <th
+                      scope="col"
+                      className="px-12 py-1 text-left text-xs font-medium text-gray-500 uppercase tracking-wider"
+                    >
```
Comment (Lehersun) [inline src/components/Pages/Parts/PartsPage.jsx:57]: Same here

File: public/index.html (hunk 4 above)
Comment (xansochi) [reply public/index.html]: I fixed it

File: public/manifest.json (hunk 5 above)
Comment (xansochi) [reply public/manifest.json]: Done

**Review by Lehersun — APPROVED** [review]
